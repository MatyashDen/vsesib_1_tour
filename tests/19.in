5 5 3 8
0 1
0 2
1 2
1 4
2 3
3 3
4 2
4 3
