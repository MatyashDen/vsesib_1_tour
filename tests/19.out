82
0LRE0DUE0LDURE0DDUUE0LDDUURE0DDDUUUE0LDDDUUURE0DDDDUUUUE0LDDDDUUUUREWWWWWWWWWWWWWW
1RLE1RRLLEWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWW
2RLE2DUE2RRLLE2RDULE2DDUUE2RRDULLE2RDDUULE2DDDUUUE2RRDDUULLE2RDDDUUULE2RRDDDUUULLE
